ballerina/tests/toXml_test.bal

```diff
@@ -1137,12 +1112,10 @@ isolated function testRecordWithNamespaceAnnotationToXml1() returns error? {
             Zip: 300,
             Country: "LK"
         },
-        'xmlns\:ns: "ns.com",
-        attr: "attr-val",
-        ns\:attr: "ns-attr-val"
```

Comment:
There is a limitation with our approach compare to the existing `xmldata` module which is we can not handle the two attributes having same `localName` but different `namespace` in the same scope. This is same for the element as well. @hasithaa  we discussed this before as well. Is this a problem since it will restrict which is allowed in the `xmldata`?